AdminClient/src/environments/enviroment.common.ts

```diff
@@ -5,5 +5,5 @@ export const commonEnvironment = {
   CANCELLATION_TEXT: 'Cancel',
   DELETE_CUSTOMER_CONFIRM_TEXT: 'Are you sure you want to delete this customer?',
   DELETE_CUSTOMER_TITLE: 'Delete Customer',
-  apiRoot: () => 'http://localhost:8080/api'
+  apiRoot: () => 'http://40.65.236.154/api'
```

Comment:
Should we leave this to be set in a config by the deployment pipeline? (Also useful for helm)

Reply:
yes that would be a good idea but that would not work for development we can have in the .prod env the enviromental variable and on the .prod this